data/mods/Dark-Skies-Above/monsters/wild_aliens/strays.json

```
@@ -41,24 +39,24 @@
       "HUMAN",
       "FILTHY",
       "BASHES",
-      "REVIVES",
       "GROUP_BASH",
-      "PUSH_MON"
+      "PUSH_MON",
+      "PATH_AVOID_FIRE"
     ]
   },
   {
     "id": "dks_mon_stray_eater",
     "type": "MONSTER",
-    "name": "hungy stray",
-    "description": "A hunger-bloated body with a distended jaw, a pitiful figure but no less hateful than its friends.  It lets out bloodcurdling howls of want and constantly regurgitates the contents of its stomach, only to seek another meal in some sort of mockery of human eating patterns.",
+    "name": { "str": "stray rockfeeder" },
+    "description": "A swollen and mishappen body pieced from the inside by growths of rocks, the lower half of its face nothing a distended jaw full of gnashing, crystalline teeth.  It lets out bloodcurdling howls of pain and constantly drips a foul smelling slurry of rocky sludge and bile.",
```

```suggestion
    "description": "A swollen and mishappen body pierced from the inside by growths of rocks, the lower half of its face nothing a distended jaw full of gnashing, crystalline teeth.  It lets out bloodcurdling howls of pain and constantly drips a foul smelling slurry of rocky sludge and bile.",
```
If this means the internal structure is meant to be biocrystal, ignore this.

---

Good catch, it's definitely supposed to be 'pierced'.